docs/products/kafka/kafka-connect/howto/s3-sink-connector-confluent.rst

```diff
@@ -0,0 +1,121 @@
+Create an S3 sink connector by Confluent
+========================================
+
+The Apache Kafka Connect® S3 sink connector by Aiven enables you to move data from an Aiven for Apache Kafka® cluster to Amazon S3 for long term storage.
+
+.. Note::
+
+    There are two versions of S3 sink connector available with Aiven for Apache Kafka Connect®: One is developed by Aiven, another developed by Confluent. This article uses the Confluent version. The S3 sink connector by Aiven is discussed in a :doc:`dedicated page <s3-sink-connector-aiven>`.
+
+Prerequisites
+-------------
+
+To setup the S3 sink connector by Confluent, you need an Aiven for Apache Kafka® service :doc:`with Apache Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`.
+
+Furthermore you need to follow the steps :doc:`to prepare the AWS account and S3 sink <s3-sink-prereq>` and collect the following information about the target S3 bucket upfront:
+
+* ``AWS_S3_NAME``: The name of the S3 bucket
+* ``AWS_S3_REGION``: The AWS region where the S3 bucket has been created
+* ``AWS_USER_ACCESS_KEY_ID``: The AWS user access key ID
+* ``AWS_USER_SECRET_ACCESS_KEY``: The AWS user secret access key
+
+Setup an S3 sink connector with Aiven CLI
+-----------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka Connect® S3 sink connector using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect® configuration file
+''''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``s3_sink.json``) with the following content:
+
+::
+
+    {
+        "name": "<CONECTOR_NAME>",
+        "topics": "<TOPIC_NAME>",
+        "connector.class": "io.confluent.connect.s3.S3SinkConnector",
+        "key.converter": "org.apache.kafka.connect.converters.ByteArrayConverter"",
+        "value.converter": "org.apache.kafka.connect.converters.ByteArrayConverter",
+        "format.class": "io.confluent.connect.s3.format.bytearray.ByteArrayFormat",
+        "flush.size": "1",
+        "s3.bucket.name": "<AWS_S3_NAME>",
+        "s3.region": "<AWS_S3_REGION>",
+        "s3.credentials.provider.class": "io.aiven.kafka.connect.util.AivenAWSCredentialsProvider",
+        "storage.class": "io.confluent.connect.s3.storage.S3Storage",
+        "s3.credentials.provider.secret_access_key": "<AWS_USER_SECRET_ACCESS_KEY>",
+        "s3.credentials.provider.access_key_id": "<AWS_USER_ACCESS_KEY_ID>"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: The connector name
```

Comment:
Any reason why we start with capital letter after ":" I think it should be  ": the connector name" Applicable for the whole list

Reply:
Also, on a minor side - some sentences have "." at the end, others - not. would be good to be consistent